docs/CRSF_SENSORS.md

```diff
@@ -0,0 +1,41 @@
+# mLRS Documentation: CRSF Sensors #
+
+([back to main page](../README.md))
+
+([back to CRSF page](CRSF.md))
+
+mLRS provides the following telemetry sensors when using CRSF:
+
+Num | Datapoint   | Description                                                             | Data Source |
+--- | ----------- | ----------------------------------------------------------------------- | ----------- |
+1   | 1RSS*       | Receiver RSSI                                                           | mLRS        |
+2   | 2RSS        | Not supported                                                           |             |
+3   | Alt         | Altitude                                                                | FC          |
+4   | ANT         | Active receiver antenna                                                 | mLRS        |
+5   | Bat%        | Battery percent remaining                                               | FC          |
+6   | Capa        | Current consumption                                                     | FC          |
+7   | Curr        | Current draw                                                            | FC          |
+8   | FM          | Flight mode                                                             | FC          |
+9   | GPS         | GPS coordinates                                                         | FC          |
+10  | GSpd        | Ground speed                                                            | FC          |
+11  | Hdg         | Heading                                                                 | FC          |
+12  | Ptch        | FC pitch angle                                                          | FC          |
+13  | RFMD        | Receiver update rate (19, 31, 1 - 50Hz)                                 | mLRS        |
```

Comment:
we could change the code so that 50 indeed shows up as 50 ... might be incompatible with some other lua stuff, I don't know
what would you say?

Reply:
Do you want to check to see if that change would break Yaapu?  I took a quick look at the code and he looks for RFMD == 1 for one of these 2Hz tasks.  https://github.com/yaapu/FrskyTelemetryScript/blob/master/OTX_ETX/c480x272/SD/WIDGETS/yaapu/main.lua#L2255

Maybe just leave alone?

Reply:
ah, ok, so lets leave it as it is :)